File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.py
class Solution:
    def getMinDistance(self, nums: List[int], target: int, start: int) -> int:
        n = len(nums)
        d = n
        for i in range(n):
            if nums[i] == target:
                cur = abs(i - start)
                if d > cur:
                    d = cur
        return d

File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.cpp
class Solution {
public:
    int getMinDistance(vector<int>& nums, int target, int start) {
        int ans;
        int n = nums.size();
        for(int i=0;i<n;i++){
            int i1 = start + i;
            int i2 = start - i;
            if(i1 >=0 && i1 < n && nums[i1] == target){
                ans = i;
                break;
            }
            if(i2 >=0 && i2 < n && nums[i2] == target){
                ans = i;
                break;
            }
        }
        return ans;
    }
};